css/components/badge.css

```diff
@@ -0,0 +1,94 @@
+@import "../helpers/spacing.css";
+@import "../utilities/variables.css";
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
```

Comment:
If you are using this same font overall in your CL, then keep this import in index.css or wherever you import all your stylesheets

Reply:
This url is for icons and I will keep this import in `index.css` from now onwards.